R/carehomes.R

```diff
@@ -509,6 +517,15 @@ carehomes_parameters_vaccination <- function(rel_susceptibility = 1) {
 }
 
 
+carehomes_parameters_waning <- function(waning_rate = 0) {
+  waning_rate <- build_waning_rate(waning_rate)
+  list(
+    # leaving this function as may add more vaccination parameters later
```

Comment:
this comment is incorrect, but can be deleted generally as there's no apology to be made here